recipes/cis/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "1.3.4" %}
+
+package:
+  name: cis
+  version: {{ version }}
+
+source:
+  fn: cis-{{ version }}.tar.gz
+  url: https://github.com/cedadev/cis/archive/{{ version }}.tar.gz
+  sha256: 319f404d9af40baf2f1a2f26c43dd82abee906d1450d9d3db8c48df5a2041ff9
+
+build:
+  number: 0
+  skip: True  # [py3k]
```

Comment:
`# [py3k]` -> `# [py3k or osx]`

This needs to be done due to the skipped `iris` build for OS X as mentioned in this issue ( https://github.com/conda-forge/iris-feedstock/issues/1 ).